engine/flink/management/src/main/scala/pl/touk/nussknacker/engine/management/FlinkDeploymentManager.scala

```diff
@@ -192,8 +129,8 @@ class FlinkDeploymentManager(
         for {
           oldJobs <- oldJobsToStop(processVersion)
           externalDeploymentIds = oldJobs
-            .sortBy(_.startTime)(Ordering[Option[Long]].reverse)
-            .flatMap(_.externalDeploymentId)
+            .sortBy(_.`start-time`)(Ordering[Long].reverse)
```

Comment:
We use now, flink's rest model. Thanks to that we don't expose this information to API (we never used these information)